$OpenBSD: patch-CMakeLists_txt,v 1.15 2017/04/18 12:08:59 landry Exp $
chunk 1: http://hub.qgis.org/issues/3999
lots of undefined refs to syms in libc, we dont link shlibs with -lc
--- CMakeLists.txt.orig	Fri Apr  7 14:12:04 2017
+++ CMakeLists.txt	Tue Apr 18 10:48:49 2017
@@ -631,12 +631,6 @@ ELSE (WIN32)
     SET(DLLEXPORT "")
   ENDIF()
 
-  IF(PEDANTIC AND NOT APPLE)
-    SET(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--no-undefined")
-    SET(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} -Wl,--no-undefined")
-    SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,--no-undefined")
-  ENDIF(PEDANTIC AND NOT APPLE)
-
 ENDIF(WIN32)
 
 ADD_DEFINITIONS("-DCORE_EXPORT=${DLLIMPORT}")
